app/views/enrollment_mailer/send_application_feedback.text.erb

```diff
@@ -0,0 +1 @@
+Votre demande d'accès à <%= @provider %> à bien été enregistrée. Notre service juridique va l'étudier. Vous serez tenu informé.
```

Comment:
Here is an email translated and adapted from kraken.com support
```
Votre demande d'accès à <%= @provider %> à bien été enregistrée.

Notre service juridique va prendre en charge votre demande au plus vite. Nous répondons habituellement sous 2 semaines, mais cela peut prendre plus longtemps lorsque nous avons beaucoup de demandes.

À noter que vous pouvez d'ores et déjà expérimenter avec les APIs de tests mis à votre disposition sur https://api.gouv.fr .

Si vous avez besoin de plus d'information vous pouvez répondre à cet email directement.

Merci pour votre patience. Nous revenons vers vous bientôt.
```

Reply:
I changed the text, I also changed the signature so it matches the sender

Reply:
I changed the email signature to match the sender's email